### DISCUSSION TITLE
Self-hosting issue

### DISCUSSION THREAD

**Discussion by samarulmeu** [giscus/giscus#169] [Q&A] [answered]
Hi!

I am trying to self host giscus and, after some work, everything works except for a small issue. When the app generates the code to include in my site, I have no ideea what variable should I change so that here `<script src="https://giscus.app/client.js` instead of `giscus.app` to have `mygiscus.app`.

Thank you!

**Comment by laymonage** [comment]
Have you deployed it somewhere? If so, just use the domain of that deployment. Otherwise, deploy it first, e.g. on [Vercel](https://vercel.com).

**Reply by samarulmeu** [reply]
Yes, I have deploy it. But still on the page the code has `giscus.app` and I have to manually change the code to get my own link giscus-test.vercel.app. You can look  [here](https://giscus-test.vercel.app).

**Reply by laymonage** [reply]
Change this line:

https://github.com/laymonage/giscus/blob/0b40c62d1d51a2e5e5e9f0674a5ad25c534c1f4a/components/Configuration.tsx#L463

Tip: you can [search the whole project for mentions of `giscus.app`](https://github.com/laymonage/giscus/search?q=giscus.app).

**Reply by samarulmeu** [reply]
Shouldn't this be done automatically? Like using a variable from `.env.local` or something?

**Reply by laymonage** [reply]
Yes, I suppose so. I guess I forgot :shrug:

**Reply by samarulmeu** [reply]
Thank you for your help and keep up the nice work!

**Reply by samarulmeu** [reply]
Hi, again! Did you manage to have a patch for this issue?
Thanks!

**Reply by laymonage** [reply]
See https://github.com/laymonage/giscus/discussions/169#discussioncomment-1350434

**Comment by laymonage** [comment, marked as answer]
Should be fixed in https://github.com/laymonage/giscus/commit/a6605bdb73a1bd40226eeec6355e46fa24d901d1, set the `NEXT_PUBLIC_GISCUS_APP_HOST` variable to e.g. `https://giscus.app`.

**Reply by samarulmeu** [reply]
Thank you so much!
